README.md

```diff
@@ -8,6 +8,26 @@ Subscription Cleanup Job is a subcomponent of [Kyma Control Plane](https://githu
 
 For more information on KCP and its components, read the [KCP documentation](https://github.com/kyma-project/control-plane/tree/main/docs) and the [KEB documentation](https://github.com/kyma-project/kyma-environment-broker/blob/main/docs).
 
+## Runnining locally
+
+- download your Gardener Kubeconfig (personal kubeconfig, service account isn't needed)
+- build subscription cleanup job: `go build ./cmd/subscriptioncleanup/main.go`
+- set environmental variables:
+  - `APP_GARDENER_PROJECT=frog-dev`
+  - `APP_GARDENER_KUBECONFIG_PATH=$PWD/kubeconfig-garden-frog-dev.yaml`
+- run the job: `./main`
+
+SCJ will look for secret bindings in the Gardener with `dirty=true` and `hyperscalerType` labels.
+If `hyperscalerType` is not set, the secret will be logged and ignored.
+If `dirty=true` is not set, the secret will be ignored.
+If there are shoots assigned to the secret bindings, it will be logged and ignored.
+
+> [!CAUTION]
+> SCJ will remove all resources associated with the secret bindings.
+> This will break any existing shoots associated with the subscription.
+>
+> Make sure nobody is using the subscription (can be a different secret) before running SCJ.
```

Comment:
How should we make sure that nobody is using the subscription?

Reply:
I checked if subscriptions assigned to our gardener have other secrets. It could be theoreticall possible that someone used the same secret in 2 gardener instances, but that would be terrible and they arguably deserve the cleanup.

Gardener shows if there are shoots using given secret.

The only possibility left is multiple secrets assigned to same subscription, that's a matter of configuring Gardener - it's not a problem on frog-gardener, as far as I was able to verify (by looking at secrets and secret bindings)